src/wavesurfer.js

```diff
@@ -378,11 +378,11 @@ var WaveSurfer = {
      *  @param  {Array}            [peaks]     Array of peaks. Required to bypass
      *                                          web audio dependency
      */
-    loadMediaElement: function (urlOrElt, peaks) {
+    loadMediaElement: function (urlOrElt, peaks, preload) {
         var url = urlOrElt;
 
         if (typeof urlOrElt === 'string') {
-            this.backend.load(url, this.mediaContainer, peaks);
+            this.backend.load(url, this.mediaContainer, peaks, preload);
```

Comment:
@katspaugh How will you do that ?
Should I delete my forked repo and created and new one ?
Or should I just close this pull request and create a simple patch ?
I don't use advanced git collaborative functions very often 🤐